### ISSUE TITLE
Longest Match segment fails when the entire input text is a full word.

### ISSUE THREAD

**Issue by ciaranbyrne** [PyThaiNLP/pythainlp#357]
**Describe the bug**
Word tokenise with longest match fails when the entire input text is a word. The problem appears to stem from the misuse of range(n) which does not include n. 
https://github.com/PyThaiNLP/pythainlp/blob/dev/pythainlp/tokenize/longest.py#L66
https://github.com/PyThaiNLP/pythainlp/blob/dev/pythainlp/tokenize/longest.py#L83

**To Reproduce**
```
from pythainlp import word_tokenize
tokens = word_tokenize('เฉียบพลัน',  engine="longest")
print(tokens)
```
The longest match for the above text is "เฉียบพลัน" which **is** present in the trie at pythainlp.tokenize.DEFAULT_DICT_TRIE.

However, the above returns ['เฉียบ', 'พลัน'].

**Expected behavior**
```word_tokenize('เฉียบพลัน',  engine="longest")```
Above code should return ['เฉียบพลัน'] since it is the longest match.

**Source Code**
In the two lines below, we use range(len_text) instead of range(len_text) + 1. This means that we never attempt to match the full input text.

https://github.com/PyThaiNLP/pythainlp/blob/dev/pythainlp/tokenize/longest.py#L66
https://github.com/PyThaiNLP/pythainlp/blob/dev/pythainlp/tokenize/longest.py#L83

**Comment by bact** [comment]
Thanks a lot to help pin point the cause of the problem.
This should be fixed.

**Comment by bact** [comment]
Fixed by #358
